rules/validator_options.go

```diff
@@ -173,3 +219,17 @@ func toAggregationTypeSet(aggTypes policy.AggregationTypes) map[policy.Aggregati
 	}
 	return m
 }
+
+func validateChars(str string, invalidChars map[rune]struct{}) error {
+	if len(invalidChars) == 0 {
+		return nil
+	}
+
+	// Validate that given string doesn't contain an invalid character.
+	for _, char := range str {
+		if _, exists := invalidChars[char]; exists {
+			return fmt.Errorf("%s contains invalid character %v", str, char)
```

Comment:
Probably want to use `fmt.Errorf("%s contains invalid character %s", str, strconv.QuoteRune(char))`? Otherwise the error message will contain the numerical value of the character instead of the human-readable string.